app/assets/stylesheets/custom.css.scss

```diff
@@ -227,13 +227,18 @@ p.copyright {
     opacity: 0;
     filter: alpha(opacity=0);
 }
+
 .q-width{
   margin: auto;
   width: 40%;
   min-width: 300px;
 }
-.row .btn{
-  margin-bottom: 1em;
+
+.btn-margin{

```

Comment:
Add spaces to css code, but btn-margin is not only for one button - its for all buttons in topic control and a have no idea how
![screenshot from 2015-12-16 18-17-54](https://cloud.githubusercontent.com/assets/15248553/11846520/a595f8ce-a421-11e5-869b-9a584e440fb8.png)
 to name it better